src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/iOS/PopupRenderer.ios.cs

```diff
@@ -116,7 +116,7 @@ protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEv
 		void CreateControl()
 		{
 			var view = Element.Content;
-			var contentPage = new ContentPage { Content = view, Padding = new Thickness(25) };
+			var contentPage = new ContentPage { Content = view, Padding = new Thickness(25), BindingContext = Element.BindingContext };
```

Comment:
Nice catch)

But what happens if Element's binding context changes during the work with popup?)

```contentPage.SetBinding(VisualElement.BindingContextProperty, new Binding { Source = Element, Path = VisualElement.BindingContextProperty.PropertyName });```

Thoughts?

Reply:
This is the way to go